Funcky.Money/Visitors/ToHumanReadableVisitor.cs

```diff
@@ -0,0 +1,48 @@
+using System.Collections.Generic;
+
+namespace Funcky
+{
+    internal class ToHumanReadableVisitor : IMoneyExpressionVisitor
```

Comment:
An immutable visitor would be really nice for this particular visitor 👀

Reply:
An Immutable Visitor would need to return the visitor on each Visit function, which is certainly possible, but I would need to change the Interface or add an immutable version of it.

https://blog.ploeh.dk/2018/08/13/a-visitor-functor/